packages/web-components/package.json

```diff
@@ -17,6 +17,7 @@
     "@agoric/assert": "^0.6.0",
     "@agoric/cache": "^0.3.2",
     "@agoric/casting": "^0.4.3-u13.0",
+    "@agoric/cosmic-proto": "0.3.0",
```

Comment:
does it work with 0.4.0?

Reply:
Following up, looks like 0.4.0 isn't published, tried `0.4.1-dev-e2e36cc.0` but doesn't seem to have the correct interface, and best not to risk https://github.com/Agoric/dapp-inter/issues/252. Will keep at 0.3.0 for now.